### ISSUE TITLE
ZenScript removeWitherForgeRecipes method does not work with Osmium

### ISSUE THREAD

**Issue by Khris777** [codetaylor/pyrotech#286]
#### Issue Description

I'm using Pyrotech and Mekanism in a modpack and I don't want Pyrotech be able to process Osmium.

I figured I had to use these four statements to remove Osmium:

```
mods.pyrotech.GraniteAnvil.removeRecipes(<mekanism:nugget:1>);
mods.pyrotech.IroncladAnvil.removeRecipes(<mekanism:nugget:1>);
mods.pyrotech.Bloomery.removeBloomeryRecipes(<mekanism:nugget:1>);
mods.pyrotech.Bloomery.removeWitherForgeRecipes(<mekanism:nugget:1>);
```

Crafttweaker log messages:

```
[POSTINITIALIZATION][CLIENT][INFO] Removing granite anvil recipes for net.minecraft.item.crafting.Ingredient@5a894c04
[POSTINITIALIZATION][CLIENT][INFO] Removing ironclad anvil recipes for net.minecraft.item.crafting.Ingredient@344224ee
[POSTINITIALIZATION][CLIENT][INFO] Removing bloomery recipes for net.minecraft.item.crafting.Ingredient@3462e521
[POSTINITIALIZATION][CLIENT][INFO] Removing bloomery recipes for net.minecraft.item.crafting.Ingredient@774751f3
```

#### What Happens

The recipes for the anvils and the bloomery are gone as expected, but the Wither Forge recipe is still there, and there is no error message in the log, so it should be gone.

#### Affected Versions

* Minecraft: 1.12.2
* Forge: 14.23.5.2847
* CraftTweaker: 1.12-4.1.20
* Dropt: 1.12.2-1.17.0
* Pyrotech: 1.12.2-1.4.22
* Athenaeum: 1.12.2-1.17.4

**Comment by codetaylor** [comment]
I'll take a look sometime. In the interim, you could try removing Osmium from the `core.compat.Ore-Custom.json` file.

Here is some more info on how Pyrotech auto-generates compat for ores: https://pyrotech.readthedocs.io/en/latest/orecompat/

**Comment by codetaylor** [comment]
Thanks for the report! Fixed in version `1.4.23`.

**Comment by Khris777** [comment]
Thanks for the advice and fix.
